config/jobs/kubernetes-sigs/scheduler-plugins/scheduler-plugins-presubmits-release-1.21.yaml

```diff
@@ -0,0 +1,55 @@
+# sigs.k8s.io/scheduler-plugins presubmits
+presubmits:
+  kubernetes-sigs/scheduler-plugins:
+  - name: pull-scheduler-plugins-verify-gofmt-release-1-21
+    decorate: true
+    path_alias: sigs.k8s.io/scheduler-plugins
+    branches:
+    - ^release-1.21$
+    always_run: true
+    spec:
+      containers:
+      - image: golang:1.16.7
```

Comment:
Shall we keep consistent with k/k?
https://github.com/kubernetes/kubernetes/tree/release-1.21/build/build-image/cross
https://github.com/kubernetes/kubernetes/tree/release-1.22/build/build-image/cross

Reply:
You mean the patch version? That should be very trivial.

Reply:
ok